### PR TITLE
chore: bump keyvault-flexvol to v0.0.12

### PR DESCRIPTION
**Pull request by aramase**
**Reason for Change**:

Update keyvault-flexvol to latest version v0.0.12

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by jackfrancis** [comment]
Could you add this new version to the `KV_FLEXVOLUME_VERSIONS` bash array in `packer/install-dependencies.sh` so that we can pre-pull it during VHD creation?

**Comment by aramase** [comment]
Thanks @jackfrancis. Done!

File: packer/install-dependencies.sh [hunk 1]
```diff
@@ -312,7 +312,10 @@ for KUBE_SVC_REDIRECT_VERSION in ${KUBE_SVC_REDIRECT_VERSIONS}; do
     echo "  - ${CONTAINER_IMAGE}" >> ${RELEASE_NOTES_FILEPATH}
 done
 
-KV_FLEXVOLUME_VERSIONS="0.0.7"
+KV_FLEXVOLUME_VERSIONS="
+0.0.7
```
Comment (ritazh) [inline packer/install-dependencies.sh]: Thanks for this PR @aramase! I don't think we need to keep the old version around. WDYT?

Comment (aramase) [reply packer/install-dependencies.sh]: Yeah, I don't think we need to keep it.

**Pull request merged**